minecraft_servers/meloncity/manifest.json

```diff
@@ -1,10 +1,34 @@
 {
   "server_name": "meloncity",
   "nice_name": "MelonCity",
-  "direct_ip": "meloncity.de",
+  "direct_ip": "proxy01.meloncity.de",
+  "supported_languages": [
+     "de"
+  ],
   "social": {
     "web": "https://meloncity.de",
+    "web_shop": "https://shop.meloncity.de",
+    "web_support": "https://wiki.meloncity.de",
     "twitter": "MelonCityDE",
-    "discord": "https://discord.gg/r2sdA4WRYn"
-  }
+    "tiktok": "meloncityde",
+    "instagram": "meloncityde",
+    "discord": "https://discord.com/invite/Y6EYzA3",
+    "youtube": "https://www.youtube.com/@meloncityde",
+    "teamspeak": "meloncity.de"
+  },
+  "discord": {
+    "server_id": 631801888253280277,
+    "rename_to_minecraft_name": true
```

Comment:
Please use `false`, as this is the default value. We will accept `true` as a value here only for verified partners.